upgrade/UPGRADE-v11.1.0.md

```diff
@@ -1,8 +1,8 @@
-# [Upgrade from v11.0.0 to v11.0.1-dev](https://github.com/shopsys/shopsys/compare/v11.0.0...master)
+# [Upgrade from v11.0.0 to v11.1.0](https://github.com/shopsys/shopsys/compare/v11.0.0...v11.1.0)
 
-This guide contains instructions to upgrade from version v11.0.0 to v11.0.1-dev.
+This guide contains instructions to upgrade from version v11.0.0 to v11.1.0.
 
-**Before you start, don't forget to take a look at [general instructions](https://github.com/shopsys/shopsys/blob/7.3/UPGRADE.md) about upgrading.**
+**Before you start, don't forget to take a look at [general instructions](https://github.com/shopsys/shopsys/blob/v11.1.0/UPGRADE.md) about upgrading.**
```

Comment:
I think that v should not be in the link - can you fix the releaser step please?

Reply:
I have done it manually... the released tag is with `v`, so the link will be all right after the release... Another option is to have the link to the branch, but that would be `master` in this case, and master may be something different in the near future 🤷